src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -93,11 +93,20 @@ class CredentialType(Enum):  # pylint: disable=too-few-public-methods
     rbac = CLOUD.endpoints.active_directory_graph_resource_id
 
 
+_GLOBAL_CREDS_CACHE = None
```

Comment:
No, a global variable is not a good idea. Can't we use a property on `application` instance?